src/tools/generate-copyright/src/cargo_metadata.rs

```diff
@@ -58,10 +54,6 @@ pub fn get_metadata_and_notices(
 ) -> Result<BTreeMap<Package, PackageMetadata>, Error> {
```

Comment:
The doc comment above this function is now outdated.

Reply:
Fixed!